tests/unit/test_metype.py

```diff
@@ -0,0 +1,42 @@
+from unittest.mock import Mock, seal
+import pytest
+
+from neurodamus.metype import BaseCell, SectionIdError
+
+def _make_mock_cell():
+    mock_cell = Mock()
+    mock_cell.soma = list(range(10))
+    mock_cell.nSecSoma = 10
+    mock_cell.axon = [10, 11]
+    mock_cell.nSecAxonalOrig = 10
+    mock_cell.dend = list(range(12, 22))
+    mock_cell.nSecBasal = 10
+    mock_cell.apic = list(range(22, 32))
+    mock_cell.nSecApical = 10
+    return mock_cell
+
+def test_get_section_id():
+    mock_cell = _make_mock_cell()
+    seal(mock_cell)
```

Comment:
I think there should still be tests for when the `nSecLastAIS / nSecNodal` don't exist.

Reply:
these 3 tests already check that no?

```
    assert BaseCell.get_section_id(mock_cell, "TestCell[0].ais[4]") == 4 * 10 + 4
    assert BaseCell.get_section_id(mock_cell, "TestCell[0].node[5]") == 4 * 10 + 5
    assert BaseCell.get_section_id(mock_cell, "TestCell[0].myelin[6]") == 4 * 10 + 6
 ```

Reply:
ahem, sorry, I mean when they exist